python/examples/integrations/Fugue_Profiling.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
+ "cells": [
```

Comment:
Can you quiet the block output to make this cleaner and easier to read? (won't show object from the last line each block)